src/Adapters/InMemoryApproverRepository.php

```diff
@@ -41,13 +41,9 @@ public function getAllCategories(): array {
 	 */
 	public function setApproverCategories( int $userId, array $categoryNames ): void {
 		$this->categoriesPerUser[$userId] = array_map(
-			fn( string $category ) => $this->normalizeCategoryTitle( $category ),
```

Comment:
It probably was never a good idea for the test double to know about normalization.